src/citrine/resources/material_spec.py

```diff
@@ -87,3 +87,18 @@ class MaterialSpecCollection(DataConceptsCollection[MaterialSpec]):
     def get_type(cls) -> Type[MaterialSpec]:
         """Return the resource type in the collection."""
         return MaterialSpec
+
+    def get_runs(self, scope: str, id: str) -> dict:
```

Comment:
The primary `get` operations for collections are defined [here](https://github.com/CitrineInformatics/citrine-python/blob/391283f46bc8cead857df4a25c04b8b319a84e7f/src/citrine/resources/data_concepts.py#L519).  What's the thought in making this a MaterialSpecCollection specific method instead of something more general?

Reply:
The methods that I'm adding methods for are specific to material templates and material specs per PLA-2138, so it didn't seem to make sense to put them on a shared type. Definitely open to discussion and others' opinions

Reply:
Just soliciting discussion.  Given how tightly scoped this is, it's the right choice.  We should probably put a ticket in to make these more general methods or to detail the full suite of appropriate routes.